rfc/0015-multi-file-jayvee/0015-multi-file-jayvee.md

```diff
@@ -65,120 +100,100 @@ valuetype MyValueType2 {
-use { MyDomainSpecificValuetype1 called Vt1} from './path/to/location.jv'; // only use the published elements from the file, access via qualified name using the alias
+use { MyDomainSpecificValuetype1 as Vt1} from './path/to/location.jv'; // only use the published elements from the file, access via qualified name using the alias
 ```
 
 References to these used elements is by their qualified name (unless altered by an alias).
 
-#### Using a library (from outside of the project)
+#### Using a package
 
 The `use` syntax is similar to importing an element of a file.
-Libraries can only be used as a whole.
+Packages can only be used as a whole.
 
 ```
-use { MyDomainLibrary } from './path/to/location.jv'; // only use the named library, access via qualified name
+use { MyDomainPackage } from './path/to/location.jv'; // only use the named package, access via qualified name
 use {
-  MyDomainLibrary1,
-  MyDomainLibrary1
-} from './path/to/location.jv'; // only use the named libraries, access via qualified name
-use { MyDomainLibrary called MyLibraryAlias} from './path/to/location.jv'; // only use the named library, access via qualified name using the alias
+  MyDomainPackage1,
+  MyDomainPackage2
+} from './path/to/location.jv'; // only use the named packages, access via qualified name
+use { MyDomainPackage as MyPackageAlias} from './path/to/location.jv'; // only use the named package, access via qualified name using the alias
 ```
 
 References to these used elements is by their qualified name (unless altered by an alias).
 
+## Decision rationale
+
+- keywords `publish` and `use` over `export` and `import` since they are less technical and better understandable for subject-matter experts
+- concept of "namespaces" is separate from publishing them for reuse in other projects (this RFC only introduces the concept of "namespaces" by the keyword `package`)
+
 ## Drawbacks
 
-- Two different sharing mechanisms (`publish` keyword, `library` concept`)
-- Elements of a pipeline cannot be reused, leading to potentially more slim pipelines and a parallel library
-- The elements of a library within a file always need the qualified name (alternative: allow access via sole name within file?)
-- The RFC does not allow re-publishing (only by putting elements into a containing library)
+- Two different sharing mechanisms (`publish` keyword, `package` concept`)
+- Elements of a pipeline cannot be reused, leading to potentially more slim pipelines and a parallel package
+- The elements of a package within a file always need the qualified name (alternative: allow access via sole name within file?)
 - Langium might not support this scoping mechanism out-of-the-box (more complex implementation)
 
 ## Alternatives
 
-- "use" syntax without braces, etc., `use MyDomainLibrary1, MyDomainLibrary2 from './path/to/file.jv';`
+- "use" syntax without braces, etc., `use MyDomainPackage1, MyDomainPackage2 from './path/to/file.jv';`
```

Comment:
We kind of defaulted to this (I assume from TypeScript experience) but I actually agree. Why have bracers? I would remove them.

Reply:
I really like the braces. 

Without it seems like some default exports (as a TS/JS-influenced dev) or different statements (`int a = 0, b = 1`). Braces clarify this imo.

Reply:
From a visual perspective, I also like the braces. Without braces, I'm unsure how very long `use` statements would be split into separate lines.